src/main/scala/karazin/scala/users/group/week1/homework/Optional.scala

```diff
@@ -0,0 +1,11 @@
+package karazin.scala.users.group.week1.homework
+
+object Optional {
```

Comment:
Please get rid of this fire. It should not be committed.

Reply:
It's Quine task. Should I add it somewhere else?

Reply:
Apply it as a text file to the Classroom task please. It's out of the context of HW1.